androminion/res/values-de/strings-server.xml

```diff
@@ -561,8 +594,20 @@
 
 	<string name="blessedVillage_receive_boon_now">Empfange %1$s (%2$s) jetzt</string>
 	<string name="blessedVillage_receive_boon_next_turn">beim Beginn des nächsten Zuges</string>
+	<string name="changeling_gain_query">%1$s: Wähle eine Karte</string>
+  	<string name="gained_card">%1$s genommen</string>
+  	<string name="changeling_exchange">%1$s in %2$s eintauschen</string>
+  	<string name="monastery_trash_copper_from_play">Entsorge ein Kupfer aus dem Spiel</string>
 	<string name="pixie_trash_for_double_boon">Entsorge dies und empfange %1$s zwei mal (2x %2$s)</string>
+	<string name="sacredGrove_receive_boon">Empfange %1$s (%2$s)</string>
+
 	<string name="boon_name_and_desc">%1$s (%2$s)</string>
 
+  	<string name="treasurer_trash_treasure">Entsorge eine Geldkarte von der Hand</string>
+  	<string name="treasurer_gain_treasure_from_trash">Nehme eine Geldkarte vom Müll auf die Hand</string>
+  	<string name="treasurer_take_key">Nimm den Schlüssel</string>
+  	
+  	<string name="innovation_set_aside_to_play">Set %1$s aside and play now</string>
```

Comment:
Looks like you missed a translation here. Would you like to update it before I merge changes?